.github/workflows/coverage.yml

```diff
@@ -0,0 +1,22 @@
+# This workflow will do a clean install of node dependencies, build the source code and run tests across different versions of node
+name: Coverage CI
```

Comment:
I think the coverage is a bad name here. Just CI or Test. Coverage is a function from tests.